tools/test/examples/examples.json

```diff
@@ -0,0 +1,8 @@
+{
+    "https://developer.mbed.org/teams/mbed/code/mbed_blinky" : ["K64F"],
```

Comment:
Blinky should build for every platform, is there a way we can remove the list here?